tests/Tags.test.tsx

```diff
@@ -150,6 +167,31 @@ describe('Select.Tags', () => {
     expect(onChange).toHaveBeenCalledWith(['a'], expect.anything());
   });
 
+  it('can render custom tags', () => {
+    const onTagRender = jest.fn();
+    const tagRender = (props: any) => {
+      const { label } = props;
+      onTagRender(label);
+      return (
+        <span className={label}>
+          {label}
+          {label}
+        </span>
+      );
+    };
+    const wrapper = mount(
+      <Select mode="tags" tokenSeparators={[',']} tagRender={tagRender} />,
+    );
+
+    wrapper.find('input').instance().focus = jest.fn();
+
+    wrapper.find('input').simulate('change', { target: { value: '1,A,42' } });
+
+    expect(wrapper.find('span.A').length).toBe(1);
+    expect(wrapper.find('span.A').text()).toBe('AA');
+    expect(onTagRender).toHaveBeenCalledTimes(3);
```

Comment:
Should check rendered element also.

Reply:
This this particular case that is why we are checking the content on the element <span/> with class "A" is "AA" (double A) since this is the custom rendering behaviour the test implements. What extra property would you want to add a test for ?